docs/types.rst

```diff
@@ -421,6 +421,23 @@ Here ``_KeyType`` can be almost any type except for mappings, a contract, or a s
 
 .. index:: !initial
 
+**********
+Builtin Constants
+**********
+
+Vyper has a few convenience constants builtin.
+
+======= ============ ==========================================
+Type    Name         Value
+======= ============ ==========================================
+address ZERO_ADDRESS 0x0000000000000000000000000000000000000000
```

Comment:
I noticed that Python's built in constants use CamelCase:
https://docs.python.org/3/library/constants.html

Reply:
We discussed this some more, and decided to leave as is.

The reason for this is because we want to reserve CamelCase names for contract interface names.

Reply:
interface names are `camelCase`, not `CamelCase`. My fault for not being more explicit about the leading capitalization.

Reply:
For now we will just go with the current "capital snake case". As this is not an uncommon convention for a python programmer ;)